### ISSUE TITLE
Suggestion Regarding New UI layout design

### ISSUE THREAD

**Issue by Sakshamgupta90** [publiclab/plots2#10892]
Hi,
I have gone through the project: refine the user interface, where we have to change the style, navbar, and many more.
But I think of something more. When I went to [publiclab.org](https://publiclab.org/), and see the UI of it, I think there is more we can add on, and many things can be replaced.

So, I started designing the layout, and finally, I have designed it. Not only the layout but many more features :)
I have attached a rough design of it.

![a1226087-63fb-4d85-b5e0-48f3d594f16d](https://user-images.githubusercontent.com/84325475/161224317-5ef890c8-fc34-4534-8054-a3f69c3471b8.jpg)

**Comment by Sakshamgupta90** [comment]
Hi,@TildaDares @jywarren @cesswairimu any suggestions?

**Comment by vincent-simon** [comment]
Hi Saksham! Love this new UI design. If u don't mind I would love to help you in the process of the new UI design. If you don't mind can we talk on Gitter about the same.

**Comment by Sakshamgupta90** [comment]
Hi Vincent! Thanks a lot :) let's wait for the mentors, for feedback and suggestions.

**Comment by on2onyekachi** [comment]
@Sakshamgupta90 This is Awesome. I like how simplified it is.

Just to be sure, I thought creating a UI design and redesigning the page is meant for the infragram project? or ,
do we get to design for PublicLab page also?.

**Comment by Sakshamgupta90** [comment]
@on2onyekachi I have design this layout for the project Refine user interface. Its our choice whether to change the layout or not. But, I think of better layout design.

**Comment by mehul89** [comment]
hello @Sakshamgupta90 it's an awsome UI user interface 🤩 I want to work with you

**Comment by Sakshamgupta90** [comment]
Hi, @mehul89 its better to take approval from the mentors :)
Thank you!

**Comment by Sakshamgupta90** [comment]
@TildaDares, this is for GSoC.

**Comment by cesswairimu** [comment]
Hi @Sakshamgupta90,  this is looking good, since this is a proposal,  you can continue refining it and once GSoC officially opens the contribution period next week,  we will open the call for proposals on the PublicLab website where you can post there along with timelines and in-depth ideas for the project and the mentors can give their input. What does this sound?  
Thanks

**Comment by Sakshamgupta90** [comment]
Okay sounds good. Thank you :)

**Comment by on2onyekachi** [comment]
> @on2onyekachi I have design this layout for the project Refine user interface. Its our choice whether to change the layout or not. But, I think of better layout design.

Oh Okay!. This layout is looking pretty good!. What app did you sketch this with? @Sakshamgupta90

**Comment by Sakshamgupta90** [comment]
@on2onyekachi  I didn't use an app for this, just simply created it on the flowchart design website: https://app.diagrams.net/

**Comment by jywarren** [comment]
Hi @Sakshamgupta90 and everyone! Yes just wanted to clarify, the Refine UI project we have listed for Outreachy and GSoC is for the Infragram project. I love the approach you're taking here and would love to see ideas related to infragram at https://github.com/publiclab/infragram/ -- thank you so much!

If you would like we can move this issue over there?

**Comment by jywarren** [comment]
Ah my apologies, i thought you were talking about https://publiclab.org/wiki/gsoc-ideas#Outreachy:+Infragram.org+full-screen+UI+design but you are looking at https://publiclab.org/wiki/gsoc-ideas#Refine+User+Interfaces+across+PublicLab.org - yes, you are correct, that is for this project. 

But, if you read the description carefully, it is a collection of narrower issues and specific interfaces that we are interested in working on, rather than the entire site UI. So we would love to see ideas related more narrowly to the issues described in there, thank you!

That said, if you wanted to do an analysis of an existing page, to highlight issues you see, and begin a discussion about whether to make broader changes to a specific page or layout, we're willing to provide feedback - it's just not the "core" of the proposal we're looking for this year. Thank you very much and I hope this is helpful!

**Comment by Sakshamgupta90** [comment]
Thanks @jywarren 
Yes I read the project description and I have decided the plans and ideas related to other issues that are mentioned as well and I will discuss it with you also. But its also written that "we are looking to improve a collection of User Interface/User eXperience (UI/UX) and related features on the platform. Some are navigational, some are stylistic, and some introduce new interfaces or ways of browsing content". So thats why I have also designed a layout for the discussion.
